data/case-studies/procivis-one_-internationally-usable-e-ids-and-credentials.json

```diff
@@ -0,0 +1,38 @@
+{
+  "headline": "Procivis One: Internationally usable E-IDs and credentials",
+  "summary": "In 2024, Procivis was selected by the U.S. Department of Homeland Security (DHS) for the Silicon Valley Innovation Program (SVIP). The project aimed to enhance global cross-border identity interoperability through open standards, including W3C Verifiable Credentials. Procivis One was successfully tested in real-world scenarios, demonstrating secure credential issuance, offline verification, and multi-standard support. The project set new benchmarks for scalable, government-ready digital identity solutions while ensuring compliance with U.S. and international regulatory frameworks. Our DHS SVIP project stands out by going beyond typical conceptual pilots or constrained test environments. By rigorously testing online and offline identity scenarios under multiple emerging global standards, our solution set a new benchmark for scalable and extensible cross-border digital identity systems.",
+  "createdAt": "2024-06-19",
+  "imageUrl": "https://cdn.prod.website-files.com/66718e3c0df645b581623cc8/66a796f09a2f56f7aa48e2c4_wide_cover_dhs_2x-p-2600.webp",
+  "url": "https://www.procivis.ch/en/case-studies/internationally-usable-e-ids-and-credentials-with-procivis-one",
+  "stage": "production",
+  "hashTags": [
+    "CrossBorderID",
+    "DigitalIDInteroperability",
+    "Interoperability",
+    "Flexibility",
+    "SelfSovereignIdentity",
+    "ProcivisOne",
+    "DHSxProcivis",
+    "SVIPProject",
+    "SSI",
+    "DecentralizedIdentity",
+    "eIDAS2.0ready"
+  ],
+  "references": [
+    "procivis-one-core",
+    "procivis-one-desk",
+    "procivis-one-verifier",
+    "procivis-one-wallet"
+  ],
+  "stakeholders": [
```

Comment:
stakeholders are not the one from the company, but more like with which other company you have realized it. So I think Homeland would be correct value